roles/ceph-common/tasks/installs/install_rh_storage_on_redhat.yml

```diff
@@ -0,0 +1,158 @@
+---
+- name: install red hat storage repository key
+  rpm_key:
+    key: "{{ ceph_stable_rh_storage_repository_path }}/RPM-GPG-KEY-redhat-release"
+    state: present
+  when:
+    - ceph_stable_rh_storage
```

Comment:
I don't think you'll need to check ``ceph_stable_rh_storage`` anywhere in this file because the when statement that includes this file already does that check.